functions/fish_right_prompt.fish

```diff
@@ -74,7 +74,7 @@ function prompt_vi_mode -d 'vi mode status indicator'
 end
 
 function fish_right_prompt -d 'Prints right prompt'
-  if test "$fish_key_bindings" = "fish_vi_key_bindings"
+  if not test "$fish_key_bindings" = "fish_default_key_bindings"
```

Comment:
I think this alters the behaviour due to the `not`, doesn't it? I would expect something like this:
```suggestion
  if test "$fish_key_bindings" = "fish_default_key_bindings"
```
Else we would have the indicator every time, even for people (like me) that do not use the feature.